src/simulations/local_elec.py

```diff
@@ -534,9 +538,8 @@ def policy_to_rewardvar(
 
         #  ----------- meet consumption + check constraints ---------------
         constraint_ok = True
-        loads, home_vars, bool_penalty = self.action_translator.actions_to_env_vars(
-            loads, home_vars, action, date, h
-        )
+        loads, home_vars, bool_penalty, flexible_q_car_action = \
```

Comment:
so here `flexible_q_car_action ` is in energy unit, right? since you set `q_car_flex = flexible_q_car_action`. In which case, I would not call it "action", as the actions as the outputs of the RL policies which have values between [-1, 1] or [0, 1]. I would call it `flexible_q_car` maybe.